app/assets/stylesheets/application.css.scss

```diff
@@ -14,3 +14,34 @@
  *= require_tree .
  *= require_self
  */
+
+$navigation-text-color: #646464;
+
+html {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Rule is empty.](https://www.codacy.com/app/conta-srdr/forsythia/file/2129633827/issues/source?bid=3139784&fileBranchId=3147671#l20)